pages/about.html

```diff
@@ -5,8 +5,31 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
+  <link href="/styles/styles-about.css" rel="stylesheet">
```

Comment:
We need to add the `..` to get the correct path. The VSCode extension doesn't catch this issue with the path. It is also true for the `images` directory.

```suggestion
  <link href="../styles/styles-about.css" rel="stylesheet">
```